suro-kafka-producer/src/test/java/com/netflix/suro/sink/kafka/TestKafkaSink.java

```diff
@@ -522,6 +522,7 @@ public void testNormailizeRoutingKey() throws Exception {
         }
         assertTrue("Leader for topic new-topic partition 0 should exist", leaderOpt.isDefined());
         final int leader = (Integer) leaderOpt.get();
+        Thread.sleep(100);
```

Comment:
unfortunately, I can't see other option. I think the test flakiness is caused by the delay in topic creation/propagation.